test/lib/ex_debug_toolbar/collector/instrumentation_collector_test.exs

```diff
@@ -19,11 +22,50 @@ defmodule ExDebugToolbar.Collector.InstrumentationCollectorTest do
     end
 
     test "it records request timeline on stop" do
-      Collector.ex_debug_toolbar(:start, %{}, %{})
-      Collector.ex_debug_toolbar(:stop, %{}, %{})
+      call_collector(&Collector.ex_debug_toolbar/3, %{})
       assert {:ok, request} = get_request(@request_id)
       assert request.data.timeline.events |> Enum.any?
       assert request.data.timeline.duration > 0
     end
   end
+
+  describe "phoenix_controller_call"  do
+    test "it starts a phoenix_controller event" do
+      Toolbar.start_request
+      Collector.phoenix_controller_call(:start, %{}, %{conn: %{}})
+      assert {:ok, request} = get_request(@request_id)
+      assert request.data.timeline.queue |> Enum.count > 0
```

Comment:
this one, i'm not sure. It depends on how much we trust ourselves :P 
are we good just asserting that if both of the methods are called in order, the event will be there in the timeline, without having to assert that one opens it, and the other closes it?